pkg/templates/templates.go

```diff
@@ -48,8 +48,7 @@ func LoadAndExecute(templateName string, funcMap template.FuncMap, data interfac
 		tplFunctions[key] = value
 	}
 
-	tplStr := templates[templateName]
-	tpl, err := template.New(templateName).Funcs(tplFunctions).Parse(tplStr)
+	tpl, err := template.New(templateName).Funcs(tplFunctions).ParseFS(content, "templates/"+templateName, "templates/common/*.xml")
```

Comment:
We can now remove the `templates` map and the `init()` method as the former's content is no longer used.